src/public/modules/forms/services/form-logic.client.service.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  BasicField,
+  LogicConditionState,
+  LogicValidConditions,
+} from '../../../../types'
+
+const conditions: LogicValidConditions[] = [
+  {
+    fieldType: BasicField.Dropdown,
+    states: [LogicConditionState.Equal, LogicConditionState.Either],
+  },
+  {
+    fieldType: BasicField.Number,
+    states: [
+      LogicConditionState.Equal,
+      LogicConditionState.Lte,
+      LogicConditionState.Gte,
+    ],
+  },
+  {
+    fieldType: BasicField.Decimal,
+    states: [
+      LogicConditionState.Equal,
+      LogicConditionState.Lte,
+      LogicConditionState.Gte,
+    ],
+  },
+  {
+    fieldType: BasicField.Rating,
+    states: [
+      LogicConditionState.Equal,
+      LogicConditionState.Lte,
+      LogicConditionState.Gte,
+    ],
+  },
+  {
+    fieldType: BasicField.YesNo,
+    states: [LogicConditionState.Equal],
+  },
+  {
+    fieldType: BasicField.Radio,
+    states: [LogicConditionState.Equal, LogicConditionState.Either],
+  },
+]
+
+const FormLogic = {
```

Comment:
suggest renaming this to `POSSIBLE_FORM_LOGIC`

Reply:
can i leave it to the next refactor, as the intention of this PR was just to test out the webpack upgrades first.

Reply:
ok, the rest lgtm